src/workloads/execution/PipelineUpdate.yml

```diff
@@ -47,193 +47,244 @@ Actors:
   - *Nop
   - *Nop
   - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+  - *Nop
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+    OnlyActiveInPhases:
+      Active: [2, 4, 6, 8]
+      NopInPhasesUpTo: 9
+      PhaseConfig:
+        Repeat: 1
 
 - Name: RunUpdates
   Type: RunCommand
   Threads: 16
   Phases:
   - *Nop
   - *Nop
-  - Repeat: 500    # Classic updates on small documents phase.
+  - *Nop
+  - Repeat: 300    # Classic updates on small documents phase.
```

Comment:
Is the repeat change deliberate?

Reply:
It's deliberate in the sense that this keeps each of the four test phases at 15 seconds, similar to the duration before. But to be honest, that line or argument does not make a lot of sense :) Increasing this back to 500 might be a good idea, since it increases the test duration by only 40 seconds in total and might even slightly increase stability. I'll change it to 500.